IsCourseActivitySyncEnabled for LearningProvider (#21464)
microsoftgraph/microsoft-graph-docs: api-reference/beta/api/learningprovider-update.md

|longLogoWebUrlForLightTheme|String|The long logo URL for the light mode that needs to be a publicly accessible image. This image would be saved to the blob storage of Viva Learning for rendering  within the Viva Learning app. Required.|
|squareLogoWebUrlForDarkTheme|String|The square logo URL for the dark mode that needs to be a publicly accessible image. This image would be saved to the blob storage of Viva Learning for rendering within the Viva Learning app. Required.|
|squareLogoWebUrlForLightTheme|String|The square logo URL for the light mode that needs to be a publicly accessible image. This image would be saved to the blob storage of Viva Learning for rendering within the Viva Learning app. Required.|

> |isCourseActivitySyncEnabled|Boolean|Indicates whether [learningCourseActivity](../resources/learningcourseactivity.md) records will be ingested for this provider or not. Default value is `false`. Learning course activities for a provider cannot be ingested if this value is false. This value needs to be updated to true when the provider wants to ingest learningCourseActivities.|

Reviewer: ```suggestion
|isCourseActivitySyncEnabled|Boolean|Indicates whether a provider can ingest [learningCourseActivity](../resources/learningcourseactivity.md). The default value is `false`. Set to true to make learningCourseActivities available for this provider.|
```
